workflow/exp/exp.jet_conus3km

```diff
@@ -1,32 +1,40 @@
 #!/user/bin/env bash
 # Aloha! This file is to setup some top level experiment options
 #
-export OPSROOT=/lfs5/BMC/nrtrr/NCO_dirs
 export NET=rrfs                    # rrfs, rtma, 
 export MESH_NAME=conus3km         # conus12km, conus3km, atl3km, etc
-export EXP_NAME=${MESH_NAME}
-export EXPDIR=/home/role.rtrr/RRFS/${VERSION}/${EXP_NAME}
-export COMROOT=${OPSROOT}/${EXP_NAME}/com      # task input and output data as well as logs
-export DATAROOT=${OPSROOT}/${EXP_NAME}/stmp    # task workdirs ($DATA) which to be removed immediately upon task completion unless KEEPDATA=yes
-export VERSION=1.0.1
-export TAG=c1v01
+export EXP_NAME=rrfsdet
+export VERSION=v2.0.1
+export TAG=c3v201
+export OPSROOT=/mnt/lfs5/BMC/wrfruc/mhu/rrfs2/${VERSION}
```

Comment:
Should the jet realtime runs be under a role account? such as 

`OPSROOT=/lfs5/BMC/nrtrr/NCO_dirs`

Reply:
Changed